deps/Makefile

```diff
@@ -30,7 +30,7 @@ CONFIGURE_COMMON += --host=$(XC_HOST)
 endif
 ifeq ($(OS),WINNT)
 ifneq ($(USEMSVC), 1)
-CONFIGURE_COMMON += LDFLAGS=-Wl,--stack,8388608
+CONFIGURE_COMMON += LDFLAGS="$(LDFLAGS) -Wl,--stack,8388608"
```

Comment:
yeah :)

previously, i've mostly just assumed that people building for windows aren't allowed to deviate from the exact directions, but it's probably good to allow more flexibility over time.